### ISSUE TITLE
Incorrect placement of markers for marker areas within scaled elements

### ISSUE THREAD

**Issue by ptrt-adriano** [ailon/markerjs2#165]
When the effective scale applied on the marker area is not 1, the markers get misplaced.

For a quick test, I have a div with a marker area functioning as expected. 

Reloading with the attribute "transform: scale(0.5);" applied to the page causes markers that would be placed at the bottom of the image to be placed half way down, markers on right top corner get placed middle top while bottom right corner get placed in the middle of the image.

It appears the mouse coordinates calculated from top left origin are getting multiplied by the scale value that the element inherits while it seemingly should be the inverse of the scale.

Dragging such markers also allows visualizing this.

**Comment by ailon** [comment]
Hi there. Thank you for reporting this. I can reproduce the issue, but do you have a practical scenario where this becomes an issue? Fixing how coordinates are calculated across the board could be a challenge and this is the first time anyone reported this. So I wonder if it's not a purely theoretical problem.

**Comment by ptrt-adriano** [comment]
Hi, the use case is as following:
I developed and released an application for annotating medical exams and drawing on them, the application was intended for tablets but after release it was demanded compatibility with phones. The application UI with the images to be edited is relatively large for a phone screen so with some scaling down and toggleability on the remainder of the UI, the main view was reduced to just the marker area and toolbar.

With that everything fit perfectly except for the markers now having this issue, which makes lower and right portions of the images not accessible.

Before this I tried setting the image dimensions smaller to fit the screen and renderAtNaturalSize set to true (The images are downloaded or compose a PDF later on so they have to be retrieved at the original size). While the renderAtNaturalSize worked as expected, this solution didn't work out as the reduced image size relative to normal marker size made the markers huge in the downloaded images. (Maybe a setting to scale markers?)
